traditions.html

```diff
@@ -36,15 +36,15 @@ <h1>Auditions</h1>
             <div class="w3-hide-medium w3-hide-large">
                 <h1>Auditions</h1>
                 <img src="images/2019/auditions_2019.jpg" class="w3-image" style="width:80%" title="Emma eats something sour">
-                <p class="w3-xlarge">We want YOU! If you're a <a href="https://www.youtube.com/watch?v=NhBktFVTjf8">virtuoso</a> or <a href="https://www.youtube.com/watch?v=V1aJOFoSR_g">just here to hang out</a>, you're invited to participate in our INTENSE audition process!<br>Auditions consist of playing board games, eating yummy snacks, making new friends, and playing a few notes if you feel like it.</p>
+                <p class="w3-xlarge">We want YOU! If you're a <a href="https://www.youtube.com/watch?v=NhBktFVTjf8">virtuoso</a> or <a href="https://www.youtube.com/watch?v=V1aJOFoSR_g">just here to hang out</a>, you're invited to participate in our INTENSE audition process!<br>Auditions consist of playing board <a href="Duck.html">games</a>, eating yummy snacks, making new friends, and playing a few notes if you feel like it.</p>
```

Comment:
So you'll notice that in a lot of the pages, text is generally repeated but with a different class so that it doesn't look weird on mobile. I'll look into how to get the pages setup so we don't have to maintain different sized copies of each paragraph, but the anchor tag will also need to be in the other copy for it to be seen on desktops.

Reply:
I thought I  switched it in both. Should be good I think